### ISSUE TITLE
Automessage matches by default?

### ISSUE THREAD

**Issue by michaelmanus** [crockpotveggies/tinderbox#4]
I just compiled/logged into my tinder account with the program and sent about 200 girls the bot's canned pickup lines.

I don't even know what to do now. On the bright side, the lines were that bad, but I have no way of carrying on conversations with that many people from all over the place. 

Maybe the bot messenger should be turned off by default?

**Comment by jamessidhu** [comment]
I agree, some poor girl has had the line about my mum showing baby pics three times. A timed delay would be great on the manual function.

**Comment by michaelmanus** [comment]
I just removed the follow up messages in FunMessages.scala. 

The pickup line arent bad but the followups repeat and rarely make sense even with that fancy sentiment analysis. 

Also, since this has been hit pretty hard by hackernews, I can imagine a lot of girls in the bay area today are gonna be asked about their preference for avocado :)

**Comment by crockpotveggies** [comment]
Sorry about that guys. I'll put a warning in the README when I get the first chance. Might be a while until I can get something more comprehensive going. The bot just had 31 forks made last night so if someone wants to open a PR I can merge it in the meantime.

**Comment by Herbstein** [comment]
Is there a way to completely turn of the the messaging bot? I want to test this, but I'm in a non-english speaking country.

**Comment by CaptainJeff** [comment]
Seems like you can just add your own messages in /tinderbox/app/models/bot/tasks/message/FunMessages.scala:

**Comment by SXRWahrheit** [comment]
Still needs a way larger variety or a better frontend for customizing messages. Once I realized what the defaults were I changed them immediately.

They got you 10 dates? I mean, I believe you, but LOL.

**Comment by jamessidhu** [comment]
You have profile information to go on after a match. How about we create pipelines which (possibly) searches the description and matches their keywords and likes to certain responses. If we don't have a message for a pipeline its flagged as manual reply where you can create a automatic pipeline so you generate custom content over time instead of everyone using the same canned responses.

**Comment by crockpotveggies** [comment]
@SXRWahrheit turns out every girl loves avocados, so that's a good start for many of them :)

@jamessidhu that's not a bad idea. I thought about formalizing the whole process a bit more. More than happy to help guide this process if people want to pitch in and improve the bot.

**Comment by Herbstein** [comment]
@CaptainJeff That works just about as well. Thank you :)

**Comment by christabella** [comment]
@crockpotveggies in the meantime you could just include a brief section in the README.md on where the user can find FunMessages.scala and modify it to their heart's content :) (I must say I'm rather sad to see "Please tell me you're a fan of moms" go)

**Comment by jamessidhu** [comment]
@crockpotveggies I'll take a look soon. Not a Scala developer to be honest so might need your eyes.

**Comment by crockpotveggies** [comment]
Added a direct link to the file in the README. Going to leave this issue open for a short while to consider adding some sort of config file for this.

@jamessidhu let me know how I can help :)

**Comment by wnstnsmth** [comment]
I think one should be able to completely disable auto messaging. This feature makes it hard for non-English speaking users to use the app "productively" ;-). @crockpotveggies , could you, in the meantime, give a hint where in the code this could be disabled? Is it enough to just change line 8 in https://github.com/crockpotveggies/tinderbox/blob/master/app/models/bot/tasks/message/FunMessages.scala to something like `def Messages = List()`?
